src/mds/MDCache.cc

```diff
@@ -448,13 +448,6 @@ void MDCache::create_mydir_hierarchy(MDSGather *gather)
     stray->store_backtrace(gather->new_sub());
   }
 
-  CInode *journal = create_system_inode(MDS_INO_LOG_OFFSET + mds->whoami, S_IFREG);
-  string name = "journal";
-  CDentry *jdn = mydir->add_primary_dentry(name, journal);
-  jdn->_mark_dirty(mds->mdlog->get_current_segment());
-
-  mydir->fnode.fragstat.nfiles++;
-  mydir->fnode.rstat.rfiles++;
```

Comment:
No issues in the fs suite from this.  Can you be more specific about assumptions elsewhere?  Hopefully nothing is assuming that the nfiles/rfiles are anything other than the number of dentries?